src/Microsoft.Health.Dicom.SqlServer/Features/Store/SqlIndexDataStoreV57.cs

```diff
@@ -0,0 +1,53 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Features.Common;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Client;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.Store;
+
+internal class SqlIndexDataStoreV57 : SqlIndexDataStoreV55
+{
+    protected static readonly Health.SqlServer.Features.Schema.Model.IntColumn TotalIndexedFileCount = new("TotalIndexedFileCount");
+    protected static readonly Health.SqlServer.Features.Schema.Model.NullableBigIntColumn TotalIndexedBytes = new("TotalIndexedBytes");
+
+    public SqlIndexDataStoreV57(SqlConnectionWrapperFactory sqlConnectionWrapperFactory)
+        : base(sqlConnectionWrapperFactory)
+    {
+    }
+
+    public override SchemaVersion Version => SchemaVersion.V57;
+
+    public override async Task<IndexedFileProperties> GetIndexedFileMetricsAsync(CancellationToken cancellationToken = default)
+    {
+        using SqlConnectionWrapper sqlConnectionWrapper = await SqlConnectionWrapperFactory.ObtainSqlConnectionWrapperAsync(cancellationToken, true);
+        using SqlCommandWrapper sqlCommandWrapper = sqlConnectionWrapper.CreateRetrySqlCommand();
+        VLatest.GetTotalAndSumContentLengthIndexedAsync.PopulateCommand(sqlCommandWrapper);
+
+        try
+        {
+            using SqlDataReader sqlDataReader = await sqlCommandWrapper.ExecuteReaderAsync(cancellationToken);
+
+            await sqlDataReader.ReadAsync(cancellationToken);
```

Comment:
Can we just use our own `ReadRow` utility? And if we use a table type I think the column properties get auto-generated.

```suggestion
            (long count, long sum) = reader.ReadRow(TotalIndexedFileCount, TotalIndexedBytes);
```